### DISCUSSION TITLE
careers page 500 error

### DISCUSSION THREAD

**Discussion by anniemossquate** [community/community#37475] [Other Feature Feedback, Questions, & Ideas]
Hello the links on the open positions section of the careers page are landing on 500 errors. Thanks :)

500 Internal Server Error
If you are the administrator of this website, then please read this web application's log file and/or the web server's log file to find out what went wrong.

**Comment by ossanna16** [comment]
Hi there @anniemossquate! Can you please elaborate a bit further on the issue and provide a screenshot?
